spring-integration-feed/src/test/java/org/springframework/integration/feed/inbound/FeedEntryMessageSourceTests.java

```diff
@@ -88,6 +88,40 @@ public void testReceiveFeedWithEntriesSorted() throws Exception {
 		assertTrue(time2 < time3);
 		assertNull(source.receive());
 	}
+	
+	// verifies that when entry has been updated since publish, that is taken into 
+	// account when determining if the feed entry has been seen before
+	@Test
+	public void testReceiveAtomFeedWithEntryHavingBeenUpdatedAfterPublishAndRepeatWithPersistentMetadataStore() throws Exception {
```

Comment:
was following the lead given by the other tests. feel free to change it to something more suitable.